_data/meltano/utilities/superset_ext/apache.yml

```
@@ -0,0 +1,150 @@
+  value: $MELTANO_PROJECT_ROOT/analyze/superset/superset_config.py
+- name: SQLALCHEMY_DATABASE_URI
+  label: SQLAlchemy Database URI
+  value: sqlite:///$MELTANO_PROJECT_ROOT/.meltano/utilities/superset/superset.db
+  description: |
+    Superset metadata database connection string. Equivalent to the `SQLALCHEMY_DATABASE_URI` setting in `superset_config.py`.
+
+    SQLite is used by default, but in production PostgreSQL or MySQL is more appropriate. See the [Superset documentation](https://superset.apache.org/docs/installation/configuring-superset/#using-a-production-metastore) for more information. To add a database driver like `psycopg2` or `mysqlclient`, refer to the [Add additional database drivers](#add-additional-database-drivers) section above.
+- name: SECRET_KEY
+  label: Secret Key
+  kind: password
+  value: thisisnotapropersecretkey
+  description: App secret key used for securely signing the session cookie and encrypting sensitive information on the database. Equivalent to the `SECRET_KEY` setting in `superset_config.py`.
+executable: superset_invoker
+commands:
+  describe:
+    executable: superset_extension
+    args: describe
+  initialize:
+    executable: superset_extension
+    args: initialize
+  invoke:
+    executable: superset_extension
+    args: invoke
+  create-admin:
+    args: create-admin
+    executable: superset_extension
+  ui:
+    executable: gunicorn
+    args: --bind $SUPERSET_UI_BIND_HOST:$SUPERSET_UI_PORT --timeout $SUPERSET_UI_TIMEOUT --workers $SUPERSET_UI_WORKERS superset.app:create_app()
+    description: Start the Superset UI. Will be available on the configured `ui.bind_host` and `ui.port`, which default to `http://localhost:8088`
+  load-examples:
+    args: load_examples
+    description: Load examples.
+logo_url: /assets/logos/utilities/superset.png
+keywords:
+- analysis
+definition: |
+ is a modern data exploration and visualization platform.
+
+ ### EDK Preview
+  This utility is in a preview stage of readiness. It is based upon our Extension Developer Kit (EDK) which is the new way to build and add plugins to Meltano Hub.
+  This preview of Superset will replace the existing Superset utility once it comes out of preview. In the meantime, we would appreciate any feedback in the [EDK repo](https://github.com/meltano/edk/) and in the [Superset Extension repo](https://github.com/meltano/superset-ext/).
+next_steps: |
+  1. If you're running Superset for the first time in a new environment you'll need to initialize the config, database, as well as generate generate a new [`SECRET_KEY`](#secret-key) to increase security:
+
+    ```sh
+    # create a new secret key
+    meltano config superset_ext set SECRET_KEY $(openssl rand -base64 42)
+    # create the initial config, database, and auth roles.
+    meltano invoke superset_ext:initialize
+    ```
+  1. Create an admin user: (use `admin` as username to be able to load the examples)
```

```suggestion
  2. Create an admin user: (use `admin` as username to be able to load the examples)
```

annoyingly it doesn't seem the markdown renders them as an ordered list